src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -9,28 +9,60 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
+import frc.robot.stateManagement.RobotStateManager;
 
 public class IntakeSubsystem extends SubsystemBase {
-
   private CANSparkMax intakeMotor;
+  private CANSparkMax chooserMotor;
 
   public IntakeSubsystem() {
     intakeMotor = new CANSparkMax(Constants.IntakeConstants.INTAKE_MOTOR_ID, MotorType.kBrushless);
+    chooserMotor =
+        new CANSparkMax(Constants.IntakeConstants.INTAKE_CHOOSER_ID, MotorType.kBrushless);
     intakeMotor.restoreFactoryDefaults();
+    chooserMotor.restoreFactoryDefaults();
+    chooserMotor.restoreFactoryDefaults();
     intakeMotor.setSmartCurrentLimit(40);
+    chooserMotor.setSmartCurrentLimit(20);
+  }
+
+  // TODO: Add check to make sure turret is below 45 degrees before running & add photogate when
+  // implemented.
+  public void runIntake() {
+    intakeMotor.set(Constants.IntakeConstants.INTAKE_PERCENTAGE);
+  }
+
+  public void reverseIntake() {
+    intakeMotor.set(-Constants.IntakeConstants.INTAKE_PERCENTAGE);
+  }
+
+  public void runChooser() {
+    chooserMotor.set(Constants.IntakeConstants.INTAKE_PERCENTAGE);
+  }
+
+  public void reverseChooser() {
+    chooserMotor.set(-Constants.IntakeConstants.INTAKE_PERCENTAGE);
+  }
+
+  public void speakerIntake() {
+    runIntake();
+    runChooser();
+  }
+
+  public void trapIntake() {
+    runIntake();
+    reverseChooser();
   }
 
-  public Command intakeCommand() {
-    // Inline construction of command goes here.
-    // Subsystem::RunOnce implicitly requires `this` subsystem.
-    return runOnce(
-        () -> {
-          /* one-time action goes here */
-        });
+  public void stopMotors() {
+    chooserMotor.set(0);
+    intakeMotor.set(0);
   }
 
-  public void setIntakeSpeed(double speed) {
-    intakeMotor.set(speed);
+  public Command intakeCommand(RobotStateManager robotStateManager) {
+    return robotStateManager.getPlacementMode().getAsInt() == 0
```

Comment:
Let's not use 0 here.  

Let's use the enum from stateManagement.PlacementMode